lib/logstash/codecs/rubydebug.rb

```diff
@@ -36,11 +35,11 @@ def encode(event)
   end
 
   def encode_default(event)
-    @on_event.call(event, event.to_hash.awesome_inspect + NL)
```

Comment:
nitpick: `awesome_inspect` still works, isn't the longer name easier to read?

Reply:
It is, I moved to `ai` in the attempt to remove all mentioned of the `awesome` in favor or `amazing`, and I noticed `amazing_inspect` wasn't added.